### DISCUSSION TITLE
Translating doc category _category_.json file

### DISCUSSION THREAD

**Discussion by Resousse** [facebook/docusaurus#8213] [Q&A] [answered]
Hello,

I've understood how to translate pages and static content, however, I have some remaining questions..
The side bar is implemented with a `_category_.json` file having `label` and `description`.
I've tried to put a file with the same name in a sub directory of `i18n/fr/docusaurus-plugin-content-docs/current/myContent`
However, it doesn't translate...

Could you tell me how to translate categories?
Thanks,

**Comment by slorber** [comment, marked as answer]
You can translate those labels using the `docusaurus write-translations` and JSON files like `docusaurus/website/i18n/en/docusaurus-plugin-content-docs/current.json`

However I can understand how this is a bit weird/confusing and not very intuitive.
Added a note about it here to see if I can improve things later: https://github.com/facebook/docusaurus/issues/4542#issuecomment-1284016559

**Comment by Resousse** [comment]
Thanks @slorber , it actually works using current.json file, thanks
